include/taskolib/Context.h

```diff
@@ -114,6 +117,7 @@ struct Context
     VariableTable variables;
 
     /// Step setup script with common functions or constants like a small library.
+    /// Overwritten when a sequence is started.
     std::string step_setup = "";
```

Comment:
Didn't we want to call this the `step_setup_script` to differentiate it from the `step_setup_function` below?

Reply:
Yep, right! Renamed to `step_setup_script`.